common/index.ts

```diff
@@ -23,25 +23,26 @@ export {
   PLUGIN_NAME,
 };
 
-export const MAP_DEFAULT_MIN_ZOOM = 0;
-export const MAP_DEFAULT_MAX_ZOOM = 22;
-export const MAP_REFERENCE_LAYER_DEFAULT_OPACITY = 100;
-export const MAP_DATA_LAYER_DEFAULT_OPACITY = 70;
-export const MAP_LAYER_DEFAULT_MIN_OPACITY = 0;
-export const MAP_LAYER_DEFAULT_MAX_OPACITY = 100;
-export const MAP_LAYER_DEFAULT_OPACITY_STEP = 1;
-export const MAP_LAYER_DEFAULT_BORDER_THICKNESS = 1;
+export const DOCUMENTS_DEFAULT_MARKER_SIZE = 5;
 export const DOCUMENTS_DEFAULT_REQUEST_NUMBER = 1000;
 export const DOCUMENTS_DEFAULT_SHOW_TOOLTIPS: boolean = false;
 export const DOCUMENTS_DEFAULT_TOOLTIPS: string[] = [];
-export const DOCUMENTS_DEFAULT_MARKER_SIZE = 5;
-export const LAYER_PANEL_SHOW_LAYER_ICON = 'eye';
 export const LAYER_PANEL_HIDE_LAYER_ICON = 'eyeClosed';
-export const MAX_LAYER_NAME_LIMIT = 35;
+export const LAYER_PANEL_SHOW_LAYER_ICON = 'eye';
+export const MAP_DATA_LAYER_DEFAULT_OPACITY = 70;
+export const MAP_DEFAULT_MAX_ZOOM = 22;
+export const MAP_DEFAULT_MIN_ZOOM = 0;
+export const MAP_LAYER_DEFAULT_BORDER_THICKNESS = 1;
+export const MAP_LAYER_DEFAULT_MAX_OPACITY = 100;
+export const MAP_LAYER_DEFAULT_MIN_OPACITY = 0;
 export const MAP_LAYER_DEFAULT_NAME = 'Default map';
-export const NEW_MAP_LAYER_DEFAULT_PREFIX = 'New layer';
-export const MIN_LONGITUDE = -180;
+export const MAP_LAYER_DEFAULT_OPACITY_STEP = 1;
+export const MAP_REFERENCE_LAYER_DEFAULT_OPACITY = 100;
+export const MAX_LAYER_LIMIT = 100; // Move this setting to Map configuration later
```

Comment:
This setting of 100 is too big for the UI. I would recommend making a default setting as 20(no great reason to support this except that it is small and will make sure that UI is responsive.)

Also, given that whatever value we choose we may endup debating what is the right default value. I would say let's get this value from config file so that customer have a way to update this value if required.

Reply:
Agree. In future this value will be part of Map settings for users to configure.

Reply:
Why we are not making this setting as of now and putting this as a part of future work?

Reply:
Map settings needs UX.

Reply:
wait there is a gap in the understanding. Let me rephrase what I was saying: I was saying we can take this settings from opensearch_dashboards.yml file.

Reply:
I don't think we need this to be part of osd yaml file. I am thinking about adding as ui settings similar to other dashboard objects. This will provide users to change this inside Dashboards instead of updating in yaml file, which requires OSD restart i believe

Reply:
I agree that it will require an OSD bootstrap and making it as a dynamic settings that can be updated using UI will be great. Please point me to the github issue which we are using to track you proposed approach.

Reply:
https://github.com/opensearch-project/dashboards-maps/issues/217